src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/CacheClearAction.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.actions;
+
+import static com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.actions.ImpactVector.Dimension.HEAP;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.AppContext;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.rca.cluster.NodeKey;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class CacheClearAction extends SuppressibleAction {
+  public static final String NAME = "CacheClear";
+
+  private final long coolOffPeriodInMillis;
+  private final boolean canUpdate;
+  private final List<NodeKey> impactedNodes;
+
+  public CacheClearAction(final AppContext appContext,
+      final long coolOffPeriodInMillis,
+      final boolean canUpdate) {
+    super(appContext);
+    this.coolOffPeriodInMillis = coolOffPeriodInMillis;
+    this.canUpdate = canUpdate;
+    this.impactedNodes = appContext.getDataNodeInstances()
+        .stream()
+        .map(ins -> new NodeKey(ins.getInstanceId(), ins.getInstanceIp()))
+        .collect(Collectors.toList());
+  }
+
+  @Override
+  public String name() {
+    return NAME;
+  }
+
+  @Override
+  public boolean canUpdate() {
+    return canUpdate;
+  }
+
+  @Override
+  public long coolOffPeriodInMillis() {
+    return coolOffPeriodInMillis;
+  }
+
+  @Override
+  public List<NodeKey> impactedNodes() {
+    return impactedNodes;
+  }
+
+  @Override
+  public Map<NodeKey, ImpactVector> impact() {
+    Map<NodeKey, ImpactVector> impactedMap = new HashMap<>();
+    impactedNodes.forEach(node -> {
+          ImpactVector impactVector = new ImpactVector();
+          impactVector.decreasesPressure(HEAP);
```

Comment:
Will this action increase `CPU` pressure? Although, since this is an SoS action, we want it to go through and not get punted due to an existing high CPU metric. So it's probably okay to not include that as an impact (also may not always increase CPU if the cache items were not really being used).